src/main/python/util/utils.py

```diff
@@ -196,8 +196,8 @@ def get_content_model(result_loc_, druid_, date_):
         headers = {
             'Content-Type': "application/json"
         }
-        url = "{}druid/v2/".format(druid_)
-        with open(Path(__file__).parent.parent.parent.parent.parent.parent.joinpath('resources', 'queries',
+        url = "{}/druid/v2/".format(druid_)
```

Comment:
This is the norm across all scripts. If you change here, then it needs to be replicated across all scripts and all API calls.

Reply:
Yes. Since the argument name is `druid_hostname`, I only decided to have the host( without **/** ) in the arguments. Please let me know if this needs to be as previous.

Reply:
Yes, the rest of the scripts are using the other standard. (with / in argument)